data/conversations.txt

```diff
@@ -10,31 +10,33 @@
-	`	It is all going smoothly until it comes time to fire up the steering engines for a test run... and you realize, much to your embarrassment, that you do not have any steering installed. Time to head back to the outfitter and fix that!`
+	`You have heard more than enough horror stories about the kinds of accidents that can happen when your starship systems malfunction and demand a non-negotiable pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of "<ship>" warming up, you run through a mental checklist. Computers: check. Life support: check.`
+	`	Everything goes as expected until it comes time to fire up the steering engines for a test run. The on-board diagnostic system gives its response and you do a double take as you check it twice. You realize that you somehow do not have any steering installed. You look around for any witnesses. With a sigh of relief, you take solace in the fact that your embarrassment is a private one as you head to the closest outfitter.`
 	
 conversation "flight check: overheating!"
 	scene "scene/engine"
-	`You've heard enough horror stories about the sort of accidents that happen when starship systems malfunction to be completely convinced of the necessity of a pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of "<ship>" warming up, you run through a mental checklist. Computers: check. Life support: check.`
-	`	By the time you reach the end of the checklist, it has grown uncomfortably warm. The air smells like ozone and hot metal, and your cameras are showing a plume of steam or smoke escaping from the hull near the engines. If your systems are running this hot while still in the atmosphere, they're likely to overheat the moment you hit hard vacuum. You are going to have to either install cooling systems, or trade out some components for lower-heat alternatives.`
+	`You have heard more than enough horror stories about the kinds of accidents that can happen when your starship systems malfunction and demand a non-negotiable pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of "<ship>" warming up, you run through a mental checklist. Computers: check. Life support: check.`
+	`	As you near the end of your checklist, the air begins to grow uncomfortably warm. As you continue working sweat drips onto your flight controls and the air grows thick with the smell of...is something burning? It reeks of ozone and hot metal, and your cameras are showing what looks like a thick gaseous cloud billowing from a bottlenecked exhaust near the engines. As you watch this, you wipe your forehead with the back of your hand and it comes away drenched. If your systems are running this hot while still in the atmosphere, they will definitely overheat the moment you hit hard vacuum. You weigh your options. You are going to have to either install cooling systems, or trade out some components for lower-heat alternatives.`
```

Comment:
This, like the other flight check conversations you've written, doesn't fit with Endless Sky's tone.

Also, again, ellipsis.

Reply:
Okidoke, I'll revise and resubmit the pull request